bridge/engine/ActiveMediaList.cpp

```diff
@@ -39,6 +39,7 @@ ActiveMediaList::ActiveMediaList(size_t instanceId,
       _maxSpeakers(audioSsrcs.size()),
       _audioParticipants(maxParticipants),
       _incomingAudioLevels(32768),
+      _incomingPttEvents(32768),
```

Comment:
I pondered this again. Perhaps it is better to add a bool ptt to AudioLevelEntry, since we report that one always from Transport. Then we need only one queue and one loop to process the audio levels and push the noiseLevel to desired value.

Reply:
Good. Going to fix that.